app/controllers/projects/samples/deletions_controller.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    # Controller actions for Project Samples Deletions
+    class DeletionsController < Projects::ApplicationController
+      before_action :sample, only: %i[new destroy]
+      before_action :new_dialog_partial, only: :new
+
+      def new
+        authorize! @project, to: :destroy_sample?
+        render turbo_stream: turbo_stream.update('samples_dialog',
+                                                 partial: @partial,
+                                                 locals: {
+                                                   open: true
+                                                 }), status: :ok
+      end
+
+      def destroy # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
+        authorize! @project, to: :destroy_sample?
```

Comment:
We already do authorization for the destroy and destroy_multiple in the services, so we don't need to have the authorizations here as well

Reply:
Fixed in df79672f6d1e1149e9876c714ed0879c6cc01590